__tests__/app.test.js

```diff
@@ -239,3 +239,212 @@ describe('GET: /api/articles/:article_id/comments', () =>
         })
     })
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+    describe('DELETE /api/comments/:comment_id', () =>
+    {
+        test('the returned body if the delete is successful should return the deleted comment from the database and the comment should no longer exist in database', () =>
+        {
+            return request(app).delete('/api/comments/2').expect(200)
+            .then(({ body }) =>
+            {
+                expect(body).toEqual
+                (
+                    {
+                        comment_id: 2,
+                        body: "The beautiful thing about treasure is that it exists. Got to find out what kind of sheets these are; not cotton, not rayon, silky.",
+                        votes: 14,
+                        author: "butter_bridge",
+                        article_id: 1,
+                        created_at: "2020-10-31T03:03:00.000Z",
+                      }
+                )
+                db.query(`SELECT * FROM comments WHERE comment_id = 2`)
```

Comment:
while this does work a better way would be to check the endpoint again instead of doing a db query in your tests